modules/reference/pages/internal-metrics-reference.adoc

```diff
@@ -123,6 +129,22 @@ Number of Kafka RPC service errors.
 
 ---
 
+=== vectorized_ntp_archiver_log_compaction_bytes_removed
```

Comment:
last minute change to the metric name

it's 
```suggestion
=== vectorized_ntp_archiver_compacted_replaced_bytes
```

the reason for the change: the metric tracks bytes removed in cloud storage by compaction, but the metric is updated independently from the garbage collection operation that will delete the old segments. 

In a healthy system, this metric will eventually match the actual deletion of files.

The name was changed to prevent a reader from thinking that when this metric increases, a delete operation was performed in cloud storage